website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -74,7 +74,7 @@ The "buildable" and "cautious" modes can be useful in environments when you're o
 
 <VersionBlock firstVersion="1.5" >
 
-There are three modes to configure the behavior when performing indirect selection (with `eager` as the default):
+There are four modes to configure the behavior when performing indirect selection (with `eager` as the default):
```

Comment:
sugg: as a docs maintenance best practice, it's better to be _generic_ to be more evergreen 

```suggestion
These are the modes to configure the behavior when performing indirect selection (with `eager` as the default):
```